### ISSUE TITLE
Probable use case if IPLD

### ISSUE THREAD

**Issue by pranavdaa** [ipld/ipld#58]
Hi i have been exploring ipld for a project integration so that i could somehow be able to find if ipld can be used to track the pinned data as to who has pinned it and for how long and related relevant info.(this is for some kind of monetization purposes on your(protocol labs)another platform

**Comment by vmx** [comment]
Yes, you an use IPLD to store additional meta information about your data.

**Comment by pranavdaa** [comment]
@vmx can i track some users data (may be encrypted) via IPLD .This is in order to monetize him for providing pinning in ipfs-cluster

**Comment by vmx** [comment]
IPLD is really just for storing your structured data. You can store whatever you like. Though this also means that it doesn't do anything automatically in regards to pinning, you need to code that yourself.

**Comment by pranavdaa** [comment]
@vmx ok thnx.Can you guid me to some of the projects using this IPLD

**Comment by vmx** [comment]
IPLD is used inside IPFS, but that's not really useful as it's mostly using it's specific Protocol Buffers based format. But you want to use CBOR. Though I'd like to point you to https://proto.school/#/blog/01. This is a tutorial about IPFS, but it explains a lot about IPLD as well.
